app/mailers/europeana/feedback_button/feedback_mailer.rb

```diff
@@ -12,7 +12,7 @@ def post(text:, type:, page:, ip:)
         @page = page
         @ip = ip
 
-        mail(to: Rails.application.config.x.feedback_mail_to, subject: 'User feedback')
+        mail(to: Rails.application.config.x.feedback_mail_to, subject: text.truncate(100))
```

Comment:
Should we be sending the `separator: ' '` option to `#truncate` to truncate on a word boundary?

Reply:
On an average feedback message and with 100 characters I don't see any big downsides to this and since it makes for slightly nicer output, sure.